setup/asciio_object/basic.pl

```diff
@@ -1,15 +1,13 @@
 use utf8 ;
 
-#TODO: QinQing please document, please separate into internal variables and use modifiable variables
-
-CANVAS_HEIGHT                  => 150,
-CANVAS_WIDTH                   => 200,
-EDIT_TEXT_INLINE               => 0,
+CANVAS_HEIGHT                  => 150,  # the height of the canvas container
+CANVAS_WIDTH                   => 200,  # the width of the canvas container
+EDIT_TEXT_INLINE               => 0,    # whether to enable inline editing
+										#     (An editing method in which the input box is embedded in the element)
 GIT_MODE_CONNECTOR_CHAR_LIST   => ['*', 'o', '+', 'x', 'X', '┼', '╋', '╬'],
-USE_CROSS_MODE                 => 0,
-USE_MARKUP_MODE                => '',
-ZOOM_LOWER_LIMIT               => 0,
-ZOOM_UPPER_LIMIT               => 28, #is it the font size? how can the limit be less than FONT_MIN and why isn't FONT_MIN used?
+										# Linker style in git mode
+USE_CROSS_MODE                 => 0,    # whether to enable crossover mode
```

Comment:
is there any reason to start in cross mode? don't we always always switch to cross mode? in that case it should be internal to asciio